2_program_structure/chess_board.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Write a program that creates a string that represents an 8×8 grid, using newline
+ * characters to separate lines. At each position of the grid there is either a space
+ * or a "#" character. The characters should form a chessboard.
+ *
+ * Passing this string to console.log should show something like this:
+ *
+ *  # # # #
+ * # # # #
+ *  # # # #
+ * # # # #
+ *  # # # #
+ * # # # #
+ *  # # # #
+ * # # # #
+ *
+ * When you have a program that generates this pattern, define a binding size
+ * = 8 and change the program so that it works for any size, outputting a grid
+ * of the given width and height.
+ */
+
+let board = '';
+let size = 8;
+
+for (let i = 0; i < size; i++) {
+    for (let j = 0; j < size + 1; j++) {
```

Comment:
Nitpick: cuando andaba leyendo esto tardé un rato en captar porqué `size + 1` jajaja. ¿Qué opinas si se reescribiera así?

```suggestion
    for (let j = 0; j <= size ; j++) {
```

Reply:
Totalmente, fue un quick fix e ignoré totalmente esa opción. Corregido 👍